<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>人民医院MICU出科登记程序1.1版</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>出科登记</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>输入数据</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>入科</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="2" column="4">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>性别:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>住院号:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="7">
               <widget class="QDateEdit" name="dateEdit">
                <property name="displayFormat">
                 <string>yyyy-MM-dd</string>
                </property>
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="7">
               <widget class="QLineEdit" name="lineEdit_3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="inputMethodHints">
                 <set>Qt::ImhDigitsOnly</set>
                </property>
               </widget>
              </item>
              <item row="0" column="6">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string> 入科时间:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>来源:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="comboBox_7">
                <property name="currentIndex">
                 <number>-1</number>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="lineEdit">
                <property name="inputMethodHints">
                 <set>Qt::ImhDigitsOnly</set>
                </property>
               </widget>
              </item>
              <item row="2" column="3">
               <widget class="QLineEdit" name="lineEdit_2">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="2" column="5">
               <widget class="QComboBox" name="comboBox_2">
                <property name="currentIndex">
                 <number>-1</number>
                </property>
                <item>
                 <property name="text">
                  <string>男</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>女</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="0" column="3" colspan="3">
               <widget class="QLineEdit" name="lineEdit_4"/>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>入科备注：</string>
                </property>
               </widget>
              </item>
              <item row="0" column="8">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="2" column="6">
               <widget class="QLabel" name="label_8">
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="text">
                 <string>年龄:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>姓名:</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>出科</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="1" column="9">
               <widget class="QCheckBox" name="checkBox">
                <property name="text">
                 <string>典型病历标记</string>
                </property>
               </widget>
              </item>
              <item row="5" column="6" colspan="3">
               <widget class="QPushButton" name="pushButton_5">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>确认</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="comboBox_5">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>备注:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1" colspan="9">
               <widget class="QLineEdit" name="lineEdit_6"/>
              </item>
              <item row="0" column="3">
               <widget class="QComboBox" name="comboBox_6">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="currentIndex">
                 <number>-1</number>
                </property>
               </widget>
              </item>
              <item row="1" column="2" rowspan="2" colspan="7">
               <widget class="QTextEdit" name="textEdit">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>150</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="5" column="0" colspan="2">
               <widget class="QPushButton" name="pushButton_4">
                <property name="text">
                 <string>清空内容</string>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QLineEdit" name="lineEdit_5"/>
              </item>
              <item row="2" column="9">
               <widget class="QPushButton" name="pushButton_6">
                <property name="text">
                 <string>编辑器</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>出科主要诊断:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="6">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string> 出科时间:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>床号:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>去向:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="9">
               <widget class="QDateEdit" name="dateEdit_2">
                <property name="displayFormat">
                 <string>yyyy-MM-dd</string>
                </property>
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>近30天内出科患者</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QListWidget" name="listWidget"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>检索</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>90</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>150</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>检索条件</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QDateEdit" name="dateEdit_3">
             <property name="displayFormat">
              <string>yyyy-MM-dd</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>至</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEdit_4">
             <property name="displayFormat">
              <string>yyyy-MM-dd</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QRadioButton" name="radioButton">
                <property name="text">
                 <string>姓名：</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_7"/>
              </item>
              <item row="1" column="0">
               <widget class="QRadioButton" name="radioButton_2">
                <property name="text">
                 <string>住院号：</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEdit_8">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QRadioButton" name="radioButton_3">
                <property name="text">
                 <string>诊断：</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="lineEdit_9">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>检索开始</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>检索结果</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QTableWidget" name="tableWidget">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>输出结果</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>903</width>
     <height>36</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>软件设置</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_2">
   <property name="text">
    <string>患者来源</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>床号设置</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>出科去向设置</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>诊断设置</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
